pages/landing/index.js

```diff
@@ -1,7 +1,7 @@
 import PropTypes from 'prop-types';
 import { getProviders, signIn, signOut, getSession } from 'next-auth/client';
 
-import Header from '../../components/Landing/Header';
+import Header from '../../components/Landing/Header/index';
```

Comment:
Do you `Header.js` would be a cleaner name? I usually think of `index` as the entry point for a page, conglomerate of functions, etc..